src/Components/MovieCard/MovieCard.jsx

```diff
@@ -1,39 +1,48 @@
-import React, { useState } from "react";
-import PropTypes from 'prop-types';
+import React, { useState, useCallback, useContext } from "react";
+import PropTypes, { func } from 'prop-types';
 
 import MovieActionsMenu from '../MovieActionsMenu';
 import MovieEdit from '../MovieEdit';
 import MovieDelete from '../MovieDelete';
+import { useToggle } from '../../Hooks/hooks';
 
 import './style.css';
+import HeaderContext from "../../Providers/HeaderContext";
 
 const MovieCard = ({ movie }) => {
```

Comment:
You render edit and delete modal windows for each card. May be it would be better to reuse modal windows for separate cards
```
            <MovieEdit movie={movie} show={isMovieEditModalShow} handleClose={toggleMovieEditModal} />
            <MovieDelete show={isMovieDeleteModalShow} handleClose={toggleMovieDeleteModal} />
```

Reply:
Yes, already noticed that and planned to move it to another place, thanks